stable/etcd-operator/values.yaml

```diff
@@ -1,59 +1,121 @@
 # Default values for etcd-operator.
 # This is a YAML-formatted file.
 # Declare variables to be passed into your templates.
-replicaCount: 1
-image:
-  repository: quay.io/coreos/etcd-operator
-  tag: v0.6.1
-  pullPolicy: IfNotPresent
-resources:
-  limits:
+
+## Install Default RBAC roles and bindings
+rbac:
+  create: true
+  apiVersion: v1beta1
+  etcdOperatorServiceAccountName: etcd-operator-sa
+  backupOperatorServiceAccountName: etcd-backup-operator-sa
+  restoreOperatorServiceAccountName: etcd-restore-operator-sa
```

Comment:
The default values for the service accounts should be `default`. They should only be used if `rbac.create=false`. The name for the service accounts your create should be base on the fullname template.

Here's an example: https://github.com/kubernetes/charts/blob/master/stable/datadog/templates/daemonset.yaml#L126

Reply:
Based on the document here: https://github.com/kubernetes/helm/blob/master/docs/chart_best_practices/rbac.md this is how it should be done? I am a bit confused?

Reply:
nm, think i understand now.

Reply:
```yaml
etcdOperatorServiceAccountName: default
backupOperatorServiceAccountName: default
restoreOperatorServiceAccountName: default
```